### PR TITLE
make agent avatar animations work again, and use them in crowds

### PR DESCRIPTION
**Pull request by howard-stearns**
test plan:
Run n instance of crowd-agent.js on a server, and summon them with the summon.js interface script.
The avatars should breath, rather than being in T-pose.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 884d22a59b94d8a5676ea20598a022cf9e082dee available at: https://gist.github.com/5ab30c88474f09220ec1eb79e45cc5a5<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8784-884d22a.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8784-884d22a.dmg

File: scripts/developer/tests/performance/summon.js [hunk 1]
```diff
@@ -23,6 +23,14 @@ var DENSITY = 0.3; // square meters per person. Some say 10 sq ft is arm's lengt
 var SOUND_DATA = {url: "http://howard-stearns.github.io/models/sounds/piano1.wav"};
 var AVATARS_CHATTERING_AT_ONCE = 4; // How many of the agents should we request to play SOUND at once.
 var NEXT_SOUND_SPREAD = 500; // millisecond range of how long to wait after one sound finishes, before playing the next
+var ANIMATION_DATA = { // T-pose until we get animations working again.
```
Comment (davidkelly) [inline scripts/developer/tests/performance/summon.js]: Should correct the comment I think

File: scripts/developer/tests/performance/summon.js [hunk 2]
```diff
@@ -23,6 +23,14 @@ var DENSITY = 0.3; // square meters per person. Some say 10 sq ft is arm's lengt
...
+    "url": "http://howard-stearns.github.io/models/resources/avatar/animations/idle.fbx",
+    //"url": "http://howard-stearns.github.io/models/resources/avatar/animations/walk_fwd.fbx",
```
Comment (davidkelly) [inline scripts/developer/tests/performance/summon.js]: Do we need the commented-out url?  Also -- do we want to put this on S3?

File: scripts/developer/tests/performance/summon.js [hunk 3]
```diff
@@ -71,18 +79,10 @@ function messageHandler(channel, messageString, senderID) {
                 rcpt: senderID,
                 position: Vec3.sum(MyAvatar.position, {x: coord(), y: 0, z: coord()}),
                 orientation: Quat.fromPitchYawRollDegrees(0, Quat.safeEulerAngles(MyAvatar.orientation).y + (turnSpread * (Math.random() - 0.5)), 0),
-                soundData: chatter && SOUND_DATA/*
+                soundData: chatter && SOUND_DATA,
                 // No need to specify skeletonModelURL
-                //skeletonModelURL: "file:///c:/Program Files/High Fidelity Release/resources/meshes/being_of_light/being_of_light.fbx",
                 //skeletonModelURL: "file:///c:/Program Files/High Fidelity Release/resources/meshes/defaultAvatar_full.fst"/,
```
Comment (davidkelly) [inline scripts/developer/tests/performance/summon.js]: Maybe get rid of this comment too?

**Comment by howard-stearns** [comment]
updated

**Comment by hifi-gustavo** [comment]
Build console output for commit: 579c95b5dd9074f3745668d244edf9faac919c17 available at: https://gist.github.com/5ab30c88474f09220ec1eb79e45cc5a5<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8784-579c95b.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8784-579c95b.dmg

**Comment by ZappoMan** [comment]
monster summoning

**Pull request merged**